src/modules/transaction/components/TxSignatureCollector/TxSignatureCollector.js

```diff
@@ -149,13 +152,19 @@ const TxSignatureCollector = ({
     );
   }
 
+  if (status && status !== 'connected') {
```

Comment:
No need to safety check existence of status, makes it more confusing imo
`if (status !== 'connected') {`

Reply:
@oskarleonard `status` will have a default value of `''` in Redux so this was the safest way to avoid triggering the wrong view especially when the component loads for the first time.